hawkbit-sdk/hawkbit-sdk-dmf/src/main/java/org/eclipse/hawkbit/sdk/dmf/amqp/VHost.java

```diff
@@ -205,7 +207,7 @@ private void handleMultiActionRequest(final Message message, final String contro
         }
     }
 
-    private void handleAttributeUpdateRequest(final Message message, final String controllerId) {
+    public void handleAttributeUpdateRequest(final Message message, final String controllerId) {
```

Comment:
I suppose the purpose of extended visibility is to allow the overriding of this method in extending classes. Then make it protected. 
I don't see a case where other (than VHost or extender) will call this method.
The same for the "to be hook" down.